### ISSUE TITLE
Accessibility - Full Report Screen - Blurry PDF Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#14]
Why is the text in the PDF along the left and bottom edge of each page blurry?

**Comment by aquilent-dgemoets** [comment]
See discussion for #5.

**Comment by jfc3-dol** [comment]
PDF text is still blurry when switching from one page to another. Think you need to look into another PDF reader for this application, because I'm assuming it still has issues of not reading any of the text with TalkBack like before.

You should see what one @skeeJay for other DOL Android applications.

**Comment by aquilent-dgemoets** [comment]
To my knowledge, @skeeJay does not work on Android applications. And none are listed on his GitHub profile page.

But as has been noted, the final version will have a hook to ask the Android phone to open the PDF using software already installed on the phone. This is the Android way of solving a problem like this, and TalkBack users will no doubt have a TalkBack-compatible PDF viewer installed on their phone.

**Comment by skeeJay** [comment]
Correct, I've never built an Android application.

**Comment by jfc3-dol** [comment]
Okay.

**Comment by aquilent-dgemoets** [comment]
The latest beta (uploaded Friday 12/4) includes the ability to have the Android phone open the PDF using software already installed on the phone (such as one that reads PDFs via TalkBack).

**Comment by nmangu** [comment]
PDF's are still blurry as one navigates it's pages (compared to iOS device view). Now, I am not sure if it is the device that is making it blurry or anything to do with the app conflict with the Android. My assumption would be the device.

**Comment by aquilent-dgemoets** [comment]
Yes, the PDF reader integrated into the app is limited.  For a better user experience, users will want to open PDFs with via the icon in the upper-right corner that will let them choose from among PDF readers installed on their phone.

**Comment by nmangu** [comment]
Ref. Issue #21 Validated.

**Comment by jfc3-dol** [comment]
Some PDFs are still blurry when viewing them in the application.

**Comment by carolynhuang** [comment]
Cannot be resolved. PDFs do not have OCR and were of accessibility complaint prior to importing in the app. This is an issue with mobile devices having small screens. To alleviate this issue, we have included the Country Webpage option so users can see all report content.

**Comment by jfc3-dol** [comment]
I was hoping an update to TalkBack might make it easier to read the PDFs for visual users since they are blurry which makes them difficult to read.

**Comment by jfc3-dol** [comment]
Closing this issue because we created a new one to easily track any progress on finding a better way to display PDF or the content a different way.

Opened issue #67 in its place.
